test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -457,6 +457,11 @@ def last_record
     assert_response :success
   end
 
+  test "should not get nonexistent status" do
+    get :get_status, params: { :id => @host.to_param, :type => 'doesnt_exist' }
+    assert JSON.parse(@response.body) == {"error" => "Status doesnt_exist does not exist."}
```

Comment:
If this test fails, it's impossible to figure out why this happens.
```suggestion
    assert_equal ({"error" => "Status doesnt_exist does not exist."}, JSON.parse(@response.body))
```